packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -389,10 +392,19 @@ export const makeVaultManager = (
     }
   };
 
+  const publicFacet = harden({
+    makeVaultInvitation: () => zcf.makeInvitation(makeVaultKit, 'MakeVault'),
+    getNotifier: () => assetNotifer,
+    getCompoundedInterest: () => compoundedInterest,
+  });
+
   /** @type {VaultManager} */
   return Far('vault manager', {
     ...shared,
     makeVaultKit,
     liquidateAll,
+    getPublicFacet: () => publicFacet,
   });
 };
+
+/** @typedef {Awaited<ReturnType<typeof makeVaultManager>>['getPublicFacet']} CollateralManager */
```

Comment:
do we need this new name?

Reply:
We have named a thing VaultManager which is worth defining,
```
/** @typedef {Awaited<ReturnType<typeof makeVaultManager>>} VaultManager */
```
Once we have that, it's not much to say `VaultManager['getPublicFacet']` where you were going to use `CollateralManager`.

Note that because it's in this file it won't be available ambiently. I think that's a good thing and we should move in that direction, but we haven't decided on that as an org. If you want it to be ambient, you can import it into `types.js` with `/** @typedef {import('./vaultManager').VaultManager} VaultManager */`. I would suggest instead using that import whenever you need the type.

Reply:
Ok so the intent is to rename VaultManager to CollateralManager? That sounds good to me and I request that either take place in this PR or be punted entirely to a different PR.